src/controllers/transfers.js

```diff
@@ -409,6 +410,8 @@ exports.putResource = function * create () {
       // version, but only allowing specific fields to change.
       transfer = updateTransferObject(originalTransfer, transfer)
     } else {
+      // If any accounts involved in a new transfer are disabled, throw error
+      yield hasDisabledAccounts(transaction, transfer)
```

Comment:
The naming of this function indicates that it returns a boolean. Might want to call it something like `validateNoDisabledAccounts`.